src/components/homePage/Book.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable quotes */
+import React from "react";
+
+const Book = () => (
+  <>
+    <div className="book">
+      <div className="left-book">
+        <span className="type">Action</span>
+        <h3>The Hunger Games</h3>
+        <span className="author">Suzanne Collins</span>
+        <ul>
+          <li>
+            <a>Comments</a>
+          </li>
+          <li>
+            <a>Remove</a>
+          </li>
+          <li>
+            <a>Edit</a>
+          </li>
+        </ul>
+      </div>
+
+      <div className="right-book">
+        <div className="progress">
+          <div className="circle">
+            <div className="inner"></div>
```

Comment:
- [ ] Kindly make sure that the book component has the two required props `title` and `author`